contrib/azurerm/roles/generate-inventory/templates/inventory.j2

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
I think it would be nice to upstream dynamic inventory generator. 
Or have the framework to add them
like: 
```
inventory/dynamic/azure.[py|sh...]
inventory/dynamic/gce.py
inventory/dynamic/aws.py
```